### DISCUSSION TITLE
No suggestion on my corporate computer (ETIMEDOUT )

### DISCUSSION THREAD

**Discussion by rotagraziosi** [community/community#7346] [Copilot Conversations]
Hello,

I am trying copilot at home on VS Code and it is working fine (and I am impressed by the results), but I am facing some issues on my corporate computer (VS Code too), I am having some timeout in the Github output window : 

`Error on ghost text request Error: connect ETIMEDOUT 51.105.161.203:443`

I am behind a proxy, maybe this causes the timeout and I didn't find a way to set a proxy settings somewhere in the configuration of Copilot.

**Comment by davecheney** [comment]
I’m sorry, the VSCode extension don’t support proxies at the moment. Can you ask your network admins to add copilot-proxy.githubusercontent.com to their allow list?
